collectors/feed/tests/test_collectors.py

```diff
@@ -0,0 +1,113 @@
+        "meta": [
+            {
+                "description": "The Packet Storm Advisory Files feed includes security advisories of various other Linux vendors."
+            },
+            {
+                "reason": "This list is a single source of all vendor advisories. Rather than subscribing to feeds of all the vendors, we use this feed."
+            },
+        ],
+    },
+    {
+        "url": "https://nodejs.org/en/feed/vulnerability.xml",
+        "name": "Node.js Vulnerability Reports",
+        "source": "internet",
+        "components": ["nodejs", "nodejs010-nodejs"],
+        "meta": [
+            {
+                "description": "The Node.js Vulnerability Reports feed contains advisories released for Node.js by the Node.js Foundation."
+            },
+            {
+                "reason": "This feed is the best source of information to learn of new Node.js vulnerabilities since we do not have a member of Product Security in the Node.js upstream community."
+            },
+        ],
+    },
+    {
+        "url": "https://feeds.feedburner.com/GoogleChromeReleases?format=xml",
+        "name": "Google Chrome Releases",
+        "source": "internet",
+        "components": ["chromium-browser"],
+        "meta": [
+            {
+                "description": "The Google Chrome Releases feed is a collection of release notes that also include security advisories. It includes advisories for various Google products; we are only interesting in the Stable Channel Chrome updates."
+            },
+            {
+                "reason": "This feed is the best source of information to learn of new Chrome vulnerabilities."
+            },
+        ],
+    },
+    {
+        "url": "https://discuss.elastic.co/c/announcements/security-announcements.rss",
+        "name": "Elastic.co Security Announcements",
+        "source": "internet",
+        "components": ["elasticsearch", "kibana"],
+        "meta": [
+            {"description": "Elastic security announcements"},
+            {
+                "reason": "Elastic components are included in OpenShift and possibly other products"
+            },
+            {"contact": "security@elastic.co"},
+            {"last_contacted": "N/A"},
+        ],
+    },
+]
+
+# Fixed subset of config.yml for smoke testing. Allows changes to config.yml without having to
+# re-record VCRs.
+FEED_CONFIG_2 = [
+    {
+        "url": "https://rss.packetstormsecurity.com/files/tags/advisory",
+        "name": "Packet Storm Advisory Files",
+        "source": "internet",
+        "meta": [
+            {
+                "description": "The Packet Storm Advisory Files feed includes security advisories of various other Linux vendors."
+            },
+            {
+                "reason": "This list is a single source of all vendor advisories. Rather than subscribing to feeds of all the vendors, we use this feed."
+            },
+        ],
+    },
+]
+
+
+@pytest.mark.django_db(True)
```

Comment:
The convention is to use the following autouse fixture, see osidb/tests/conftest.py:

```python
@pytest.fixture(autouse=True)
def enable_db_access_for_all_tests(db):
    pass
```

Reply:
Done. Thank you.